<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registering</class>
 <widget class="QDialog" name="registering">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="register_2">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>533</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Script MT Bold</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="role">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>131</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Forte</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>What is your role?</string>
   </property>
   <widget class="QRadioButton" name="doc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Doctor</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="admin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="nurse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nurse</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="patient">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Patient</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="errorregister">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>540</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Script MT Bold</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;* All fields must be filled&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Forte</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Re-type password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="repass"/>
    </item>
    <item>
     <widget class="QLabel" name="repasserror">
      <property name="font">
       <font>
        <family>Script MT Bold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;* Not Matching&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Forte</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passedit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Forte</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameedit"/>
    </item>
    <item>
     <widget class="QLabel" name="errorname">
      <property name="font">
       <font>
        <family>Script MT Bold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;* Already Existing&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="gender">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Forte</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>What is your gender?</string>
   </property>
   <widget class="QRadioButton" name="male">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="female">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Forte</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ageedit"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="days">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Forte</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Choose your working days</string>
   </property>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Sunday</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Tuesday</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Thursday</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Saturday</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Wednesday</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Monday</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="time">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>290</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Forte</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Write your working time</string>
   </property>
   <widget class="QLineEdit" name="toedit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fromedit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>To</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>From</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_speciality">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>370</y>
     <width>241</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 14pt &quot;Segoe UI&quot;;
font: 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="title">
    <string>Speciality</string>
   </property>
   <widget class="QRadioButton" name="radioButton_Nutrition">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Nutrition</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_OG">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Obstetrics and Gynecology</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Ophthalmology">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ophthalmology</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_internalmedicine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Internal Medicine</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Dermatology">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dermatology</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
